src/constructor.cpp

```diff
@@ -553,83 +553,93 @@ int main(int nargs, char **argv)
     parse_variables(convert_string(file_storage_location+"variables"));
     unsigned long t_usec = walltime_usec();
 
-    if(brownfield)
-        square_coordinate = get_square_coordinate(get_existing_reservoir(arg1));
+    pairs = read_rough_pair_data(convert_string(file_storage_location+"processing_files/pretty_set_pairs/"+fname+"_rough_pretty_set_pairs_data.csv"));
 
-    if (pit)
-        pit_details = get_pit_details(arg1);
+    uint total_pairs = 0;
+	for(uint i = 0; i<pairs.size(); i++)
+		total_pairs += pairs[i].size();
 
-    BigModel big_model = BigModel_init(square_coordinate);
-    vector<string> country_names;
-    vector<vector<vector<GeographicCoordinate>>> countries = read_countries(file_storage_location+"input/countries/countries.txt", country_names);
+	if (total_pairs > 0) {
 
```

Comment:
To avoid a large nested region, could we just use
```
if (total_pairs == 0){
    printf("Constructor finished for %s. Found no pairs. Runtime: %.2f sec\n", convert_string(fname), 1.0e-6*(walltime_usec() - t_usec) );
    return;
}